htdocs/src/Application/Grossum/MenuBundle/Resources/config/doctrine/Menu.orm.yml

```diff
@@ -0,0 +1,11 @@
+Application\Grossum\MenuBundle\Entity\Menu:
+    type: entity
+    table: menu
```

Comment:
Add issue for it. When we will refactor bundels we add this part
https://github.com/GrossumUA/Symfony2-DevBox/issues/4